sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -129,6 +129,43 @@ VALUES ('page_validation','add_edit_event#theform_groups','/interface/main/calen
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'common#new-encounter-form', '/interface/forms/newGroupEncounter/common.php', 160, '{pc_catid:{exclusion: ["_blank"]}}', 1);
 #EndIf
 
+-- 1. users
+--
+#IfMissingColumn users weno_prov_id
+ALTER TABLE `users` ADD `weno_prov_id` VARCHAR(15) DEFAULT NULL;
+#Endif
+
+-- 2. prescriptions 
+--
+#IfMissingColumn prescriptions ntx 
+ALTER TABLE `prescriptions` ADD `ntx` INT(2) DEFAULT NULL;
+#Endif
+
+-- 3. prescriptions 
+--
+#IfMissingColumn prescriptions rtx 
+ALTER TABLE `prescriptions` ADD `rtx` INT(2) DEFAULT NULL;
+#Endif
+
+-- 4. prescriptions 
+--
+#IfMissingColumn prescriptions txDate 
+ALTER TABLE `prescriptions` ADD `txDate` DATE NOT NULL;
```

Comment:
Question, If a clean install is being done then these tables will have these entries, right? Also, I don't see any other statements like these in the database.sql file.